### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by wifi75** [deiger/AirCon#19]
From mqtt explorer there are all command.. and if i change mode from remote controll on mqtt explorer change correctly... but from HA not work maybe the config it is not correct?

see below

![Cattura](https://user-images.githubusercontent.com/31816231/83261756-2848ba80-a1bc-11ea-823d-610ff2dd471f.PNG)

only temperature work....
![hik](https://user-images.githubusercontent.com/31816231/83261910-69d96580-a1bc-11ea-833c-a365a0d95516.PNG)

**Comment by ck007max** [comment]
The problem is that hassio can't identify the status of the AC because the value topic is written in capitol letter. The author of the script could transform the topic in a way that hassio can use for update the status and for the command. Infact the only function that works is a number status.

**Comment by wifi75** [comment]
Hi @ck007max  my topic it is lowecase: 
see below:

./hisense.py --port 8888 --ip 192.168.1.251 --config config_stanzetta.json --mqtt_host 192.168.1.52 --mqtt_user user:password --mqtt_topic hisense_stanzetta

**Comment by deiger** [comment]
The data provided to HA is always in lower case, see the explanation in #15.
@wifi75, which configuration are you using for HA? If you are using [mine](../blob/master/hass/configuration.yaml), and change the mqtt_topic, you should also change the prefix for all the topics in the configuration.

**Comment by ck007max** [comment]
I use your config and your original topic, but when I launch the hisense.py script I receive these following messages from HA: 2020-05-29 20:15:29 ERROR (MainThread) [homeassistant.components.mqtt.climate] Invalid swing_modes mode: 
2020-05-29 20:15:29 ERROR (MainThread) [homeassistant.components.mqtt.climate] Invalid fan_modes mode: 
2020-05-29 20:15:31 ERROR (MainThread) [homeassistant.components.mqtt.climate] Invalid modes mode: 
So I think that there is a problem with HA and mqtt topic

**Comment by deiger** [comment]
IIUC, the error message for e.g. "Invalid fan_modes mode:" should end with the invalid mode. It seems that it got an empty string...
I think I understand the issue. Try removing the value_template row from the config. The response should not be escaped as JSON.

**Comment by wifi75** [comment]
@deiger Of course I have already done it but it doesn't work  see below:
![1](https://user-images.githubusercontent.com/31816231/83298946-9fe70b80-a1f5-11ea-9e72-e94da6903b43.PNG)
very strange because in mqtt explorer there is all the data!
![2](https://user-images.githubusercontent.com/31816231/83299348-49c69800-a1f6-11ea-9b26-b2eaf2cf167e.PNG)

**Comment by wifi75** [comment]
@deiger Read also this message afgter status service command:

![3](https://user-images.githubusercontent.com/31816231/83299837-38ca5680-a1f7-11ea-88cb-2011d045e7a4.PNG)

**Comment by deiger** [comment]
I see in your screenshot value_message is set. Could you try removing that line?

**Comment by wifi75** [comment]
@deiger  I have tryed to remove that line but doesn't work

**Comment by ck007max** [comment]
@deiger I've followed your change and I've removed "value_template". Now, the HACV componet read the status, but there is a problem with the command line. I can't change the status of the settings via HA. If I push "cool" button, nothing happens. And there is a problem with fan_only mode: 2020-05-30 08:29:45 ERROR (MainThread) [homeassistant.components.mqtt.climate] Invalid fan_modes mode: fan_only 
![Immagine](https://user-images.githubusercontent.com/66070211/83321581-eae23c80-a250-11ea-9a7e-801dd85ff28c.png)
